framework/src/bcs/ADDirichletBCBase.C

```diff
@@ -8,35 +8,20 @@
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 
 #include "ADDirichletBCBase.h"
+#include "MooseVariableFE.h"
+#include "SystemBase.h"
+#include "libmesh/node.h"
```

Comment:
why do you need more headers with just deleted lines

Reply:
to challenge the compiler 😆